Vagrantfile

```diff
@@ -105,6 +108,9 @@ Vagrant.configure("2") do |config|
     app.vm.network "private_network", ip: ENV.fetch("MMW_APP_IP", "33.33.34.10")
 
     app.vm.synced_folder "src/mmw", "/opt/app"
+    # Facilitates the sharing of Django media root directories across virtual machines.
+    app.vm.synced_folder ".vagrant/machines/app/virtualbox/media", "/var/www/mmw/media",
```

Comment:
Similar, except here two things are different:

- `MEDIA_ROOT` is at a different location
- We are mounting the directory with `dmode=777` to make it easy for Gunicorn to write to it